Train batch 0: Loss --- 0.6917227506637573
Train batch 1: Loss --- 0.664630115032196
Train batch 2: Loss --- 0.6802591681480408
Train batch 3: Loss --- 0.641568124294281
Train batch 4: Loss --- 0.6415528059005737
Train batch 5: Loss --- 0.604295015335083
Train batch 6: Loss --- 0.6214702725410461
Train batch 7: Loss --- 0.6196180582046509
Train batch 8: Loss --- 0.5761945247650146
Train batch 9: Loss --- 0.5576255917549133
Train batch 10: Loss --- 0.5513362884521484
Train batch 11: Loss --- 0.5503718852996826
Traceback (most recent call last):
  File "tools/train.py", line 34, in <module>
    train_model(epochs, criterion, optimizer)
  File "tools/train.py", line 20, in train_model
    for batch_idx, (data, labels) in enumerate(train_loader):
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 49, in fetch
    data = self.dataset.__getitems__(possibly_batched_index)
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torch/utils/data/dataset.py", line 399, in __getitems__
    return [self.dataset[self.indices[idx]] for idx in indices]
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torch/utils/data/dataset.py", line 399, in <listcomp>
    return [self.dataset[self.indices[idx]] for idx in indices]
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 229, in __getitem__
    sample = self.loader(path)
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 268, in default_loader
    return pil_loader(path)
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 248, in pil_loader
    return img.convert("RGB")
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/PIL/Image.py", line 922, in convert
    self.load()
  File "/home/muneeb/Desktop/zuha/.env/lib/python3.8/site-packages/PIL/ImageFile.py", line 291, in load
    n, err_code = decoder.decode(b)
KeyboardInterrupt